android/src/main/AndroidManifest.xml

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.guichaguri.trackplayer">
+          package="com.doublesymmetry.trackplayer">
 
     <uses-permission android:name="android.permission.WAKE_LOCK" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
 
     <application>
 
         <!-- The main service, handles playback, playlists and media buttons -->
-        <service android:name=".service_old.MusicService" android:enabled="true" android:exported="true">
+        <service android:name="com.doublesymmetry.trackplayer.service.MusicService" android:enabled="true" android:exported="false">
```

Comment:
did you change the `exported` to false on purpose?

Reply:
Yes. We don't want other random apps to use our service.

Reply:
aha, got it. oki.